Supervisor feedback system evaluation (#96)
chhoumann/p8: report/src/system_evaluation/results.tex

\section{Results}\label{sec:experiment_results}
%describe in terms of criteria for using bluetooth. `'
Having created a map that can be used to classify received measurements and having collected information about these classifications, we will now summarizes and comment on the results. 
The classification results of the experiments were gathered in the locations shown on Figure \ref{fig:experiment_setup}. During the experiment, k-values of $2$, $3$, and $4$ were used.  

> The results of the experiment can be seen in Tables \ref{lst:resultsPhone}, \ref{lst:resultsPhoneS20}, and  \ref{lst:resultsTablet}.

Reviewer: ```suggestion
The results of the experiment can be seen in Tables \ref{lst:resultsPhone}, \ref{lst:resultsPhoneS20}, and \ref{lst:resultsTablet}.
```

> The tables shows the number of correct classifications. 

Reviewer: ```suggestion
These tables show the number of correct classifications. 
```

    3                             & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{7/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{5/10} & \multicolumn{1}{l|}{5/10} & \multicolumn{1}{l|}{5/10} & \multicolumn{1}{l|}{8/10} & 8/10                   \\ \hline
    4                             & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{9/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{6/10} & \multicolumn{1}{l|}{6/10} & \multicolumn{1}{l|}{7/10} & \multicolumn{1}{l|}{9/10} & 10/10                  \\ \hline
    \end{tabular}}
    \caption{Evaluation results from Lenovo TabM10 FHD Plus showing the number of correct classifications out of a possible ten for each $k$-value}
    \label{lst:resultsTablet}
\end{table}

> On Table \ref{lst:resultsPhone_precision} see that a lower $k$-value results in a larger uncertainty in the classification.

Reviewer: ```suggestion
On Table~\ref{lst:resultsPhone_precision}, we see that a lower $k$-value results in a greater uncertainty in the classifications.
```

> However, it is worth noticing that for measuring points \textit{B} and \textit{J}  (see Figure \ref{fig:experiment_setup}) a large uncertainty in classification is observed.

Reviewer: Right?

With these results, it's clearer why you used `B` and `J` as reference earlier - but we didn't know that at the time, and so won't the readers.

```suggestion
However, it is worth noticing that for classification points \textit{B} and \textit{J}, shown in Figure~\ref{fig:experiment_setup}, a large uncertainty in classification is observed.
```

> We suspect that this is a consequence of the glass pane next to the door having a much lower absorbation of radio frequencies than the door and the wall next to it. 

Reviewer: Need cite?
That's a nice claim. How about you back it up with a source
```suggestion
We suspect that this is caused by the glass pane next to the door having a much lower absorbation of radio frequencies than the door and the wall next to it. 
```

Author: That is why I have written suspect. I have not been able to find any paper describing the effect of glass vs regular wall, or just glass for that matter.

Reviewer: Suspect doesn't, when I read the sentence, imply our belief that glass has lower absorption of radio frequencies, but rather that we suspect the low absorption is the cause - which we are not able to provide evidence for.

This low absorbation results in the two data points next to the pane having similar distances to both map values labelled as inside and some as outside, therefore giving uncertain classifications.

> Similarly, for all devices, when performing the classification in locations H, I, J, K, and L shows, the results are vastly different from the other locations.

Reviewer: Did we interpret this right? But probably rephrase, it isn't a clear sentence.

```suggestion
Similarly, for all devices, when performing classification in locations H, I, J, K, and L, the results are vastly different from the other locations.
```

These locations are all outside of the room. 
We suspect that this can be due to two things.

> First and foremost, it might be due to the created map being unreliable. 

Reviewer: ```suggestion
First, it might be due to the map being unreliable. 
```

When creating the map, the wall separating the room and the common area might cause large fluctuations in the strength of the received signal. 

> Therefore, classification using the map becomes unreliable.

Reviewer: ```suggestion
Consequently, classifications using the map can become unreliable.
```

> We tried to accomodate this by enforcing all map entries to have a standard deviation of 6.5 \todo{units and 6.5}. 

Reviewer: What was that todo about???

Also: explain why sd of 6.5 was chosen in the first place.

```suggestion
We tried to accomodate this by enforcing all map entries to have a standard deviation of $6.5$dBm. 
```

> Enforcing a lower standard deviation of the map might alleviate these issues.

Reviewer: OK, but why? And if so, why didn't ours? Is ours too high?

> However, this will increase the time it takes to create the map, as more samples will be required if the signal fluctuates.

Author: It is one of the things we base the selection of technology on. 
![image](https://github.com/chhoumann/p8/assets/32592615/d240ffcc-7752-4072-8d3c-f7551f6e49bf)

> However, this will increase the time it takes to create the map, as more samples will be required if the signal fluctuates.

Author: It would be weird to disregard it entirely.

Furthermore, detecting these signal outliers might be difficult if the signal often fluctuates by high degrees.      

> Secondly, by having placed beacons only inside of the meeting room, and by having more classification data representing locations inside the meeting room, the classification results might be skewed towards \textit{inside} classifications. 

Reviewer: Shouldn't this be closer to `first`? Otherwise rephrase. They're too far apart.
```suggestion
Secondly, by having placed beacons only inside of the meeting room, and by having more classification data representing locations inside the meeting room, the classification results might be biased towards \textit{inside} classifications. 
```

Author: ```suggestion
Furthermore, by having placed beacons only inside of the meeting room, and by having more classification data representing locations inside the meeting room, the classification results might be biased towards \textit{inside} classifications. 
```

Reviewer: Readers would probably wonder where the second thing went when we say "two things" followed by "first" and no "second".

> One could try lessen the effect of this skewing by placing one or more beacons in the common area, and treat signals from this beacon as a weight towards a classification of \textit{outside}.  

Reviewer: The weight thing: that's not how it works.
Also, explain why it might help.

```suggestion
One could try to lessen the effect of this bias by placing one or more beacons in the common area.
```

Author: We need to explain why this resolves the bias.

Reviewer: Not sure why we think it would.

Trying to steelman:
- Fluctuations are due to beacons being behind the wall, and therefore placing beacon(s) outside the room helps stabilize signal strengths

But the walls & their material are constants. Would it really make a difference? Why wouldn't it just be consistently bad signal, and therefore make no difference?

I think an effort to limit confounding variables, e.g. by keeping as many variables constant as we can, would show an improvement. Perhaps also placing an equivalent amount of grid points inside & outside?

    \label{lst:resultsTablet}
\end{table}

> On Table~\ref{lst:resultsPhone_precision}, we see that a lower $k$-value results in a greater uncertainty in the classifications.

Reviewer: ```suggestion
In Table~\ref{lst:resultsPhone_precision}, we see that a lower $k$-value results in a greater uncertainty in the classifications.
```

However, it is worth noticing that for classification points \textit{B} and \textit{J}, shown in Figure~\ref{fig:experiment_setup}, a large uncertainty in classification is observed.
We suspect that this is caused by the glass pane next to the door having a much lower absorbation of radio frequencies than the door and the wall next to it. 
This low absorbation results in the two data points next to the pane having similar distances to both map values labelled as inside and some as outside, therefore giving uncertain classifications.

> Similarly, for all devices, when performing classification in locations H, I, J, K, and L, the results are vastly different from the other locations.

Reviewer: ```suggestion
Similarly, for all devices, when performing classification in locations \textit{H}, \textit{I}, \textit{J}, \textit{K}, and \textit{L}, the results are vastly different from the other locations.
```

These locations are all outside of the room. 
There are a couple of reasons we suspect for this. 

The map's unreliability might be a contributing factor. During the map's construction, the wall between the room and the common area might have triggered significant fluctuations in the received signal's strength.
This inconsistency could potentially lead to unreliable classifications using the map. In an attempt to remedy this, we regulated all map entries to possess a standard deviation of 6.5.
Reducing the standard deviation of the map might help to mitigate these issues.
However, this process could extend the time required to create the map, as more samples would be needed in the face of fluctuating signals.
Additionally, identifying these signal outliers could be challenging if the signal is highly volatile.

Furthermore, the placement of beacons only within the meeting room and a larger amount of classification data representing locations within the meeting room may lead to a bias towards \textit{inside} classifications. 

> To counter this bias, one possible approach is to position one or more beacons in the common area, considering signals from this beacon as a bias towards an 'outside' classification.

Reviewer: ```suggestion
To counter this bias, one possible approach is to position one or more beacons in the common area, considering signals from this beacon as a bias towards an \textit{outside} classification.
```
